code/lib/cli/src/automigrate/fixes/mdx-1-to-2.ts

```diff
@@ -47,11 +60,23 @@ export const mdx1to2: Fix<Mdx1to2Options> = {
     `;
   },
 
-  async run({ result: { storiesMdxFiles }, dryRun }) {
-    await Promise.all(
-      storiesMdxFiles.map(async (fname) => {
+  async run({ result: { storiesMdxFiles, nonStoriesMdx1Files }, dryRun }) {
+    await Promise.all([
+      ...storiesMdxFiles.map(async (fname) => {
+        const contents = await fse.readFile(fname, 'utf-8');
+        const updated = fixMdxComments(fixMdxStyleTags(contents));
+        if (updated === contents) {
+          logger.info(`🆗 Unmodified ${basename(fname)}`);
+        } else {
+          logger.info(`✅ Modified ${basename(fname)}`);
+          if (!dryRun) {
+            await fse.writeFile(fname, updated);
+          }
+        }
+      }),
+      ...nonStoriesMdx1Files.map(async (fname) => {
```

Comment:
@ndelangen My idea is to run this on the storiesMdxFiles only and we assume that non-`.stories.mdx` files are already valid MDX2. Can discuss on a call if it's more convenient.

Reply:
Does it hurt running this over normal MDX files?

Reply:
Only in that it's inconsistent with the check. Automigrations run every time you upgrade for now, so it would be nice if there was a way to make them stop if there are no more `.stories.mdx` files, which is what the check currently does even with your revision. To me, if the automigration is triggered by having `.stories.mdx` files, then it makes more sense to run the migration on those `.stories.mdx` files.

Reply:
@ndelangen whatever you decide, I'd suggest applying the same logic to all sub-fixes that we add to this automigration. since this is about migrating from mdx1 to 2 and that should equally apply to any MDX file that needs to be migrated (the logic you added only does the comments for the non-stories.mdx files but why not the styles also?)

Reply:
Let's chat about this tomorrow?